src/test/java/com/marizueva/laboratory/hw4/voidpageobject/tests/FailedSiteTestWithHardAssertions.java

```diff
@@ -0,0 +1,104 @@
+package com.marizueva.laboratory.hw4.voidpageobject.tests;
+
+import com.marizueva.laboratory.hw4.testdata.UsedInTestTerms;
+import com.marizueva.laboratory.hw4.utils.testnaming.FeaturesNaming;
+import com.marizueva.laboratory.hw4.utils.testnaming.StoriesNaming;
+import com.marizueva.laboratory.hw4.voidpageobject.pages.HeaderItems;
+import com.marizueva.laboratory.hw4.voidpageobject.pages.HomePage;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class FailedSiteTestWithHardAssertions extends BaseTest {
+
+    @Test
+    @Story(value = StoriesNaming.JDI_SITE)
+    @Feature(value = FeaturesNaming.HOME_PAGE)
+    public void testBrowserTitleForHome() {
+        homePage = new HomePage(driver);
+        homePage.openSite();
+        Assert.assertEquals(driver.getTitle(), homePageTitle);
+
+
+        homePage.performLogin(userName, password);
+        String loggedUserName = homePage.getLoggedUserName();
+
+        Assert.assertEquals(loggedUserName, loggedUserName);
+
+
+        String headerItemHome = homePage.getHeaderItem(HeaderItems.HOME);
+        Assert.assertEquals(headerItemHome, HeaderItems.HOME.getRequestedTabText());
+
+        String headerItemContactForm = homePage.getHeaderItem(HeaderItems.CONTACT_FORM);
+        Assert.assertEquals(headerItemContactForm, HeaderItems.CONTACT_FORM.getRequestedTabText());
+
+        String headerItemService = homePage.getHeaderItem(HeaderItems.SERVICE);
+        Assert.assertEquals(headerItemService, HeaderItems.SERVICE.getRequestedTabText());
+
+        String headerItemMetalAndColors = homePage.getHeaderItem(HeaderItems.METAL_AND_COLORS);
+        Assert.assertEquals(headerItemMetalAndColors,
+                HeaderItems.METAL_AND_COLORS.getRequestedTabText());
+
+
+        boolean benefitIcon1 = homePage.isBenefitIconDisplayed(1);
+        Assert.assertTrue(benefitIcon1);
+
+        boolean benefitIcon2 = homePage.isBenefitIconDisplayed(2);
+        Assert.assertTrue(benefitIcon2);
+
+        boolean benefitIcon3 = homePage.isBenefitIconDisplayed(3);
+        Assert.assertTrue(benefitIcon3);
+
+        boolean benefitIcon4 = homePage.isBenefitIconDisplayed(4);
+        Assert.assertTrue(benefitIcon4);
+
+
+        boolean benefitText1 = homePage.isBenefitTextDisplayed(1);
+        Assert.assertTrue(benefitText1);
+
+        boolean benefitText2 = homePage.isBenefitTextDisplayed(2);
+        Assert.assertTrue(benefitText2);
+
+        boolean benefitText3 = homePage.isBenefitTextDisplayed(3);
+        Assert.assertTrue(benefitText3);
+
+        boolean benefitText4 = homePage.isBenefitTextDisplayed(4);
+        Assert.assertTrue(benefitText4);
```

Comment:
Could be combined to the loop for the collecting text and after that you could comparing two collections with text

Reply:
fixed